devito/types/basic.py

```diff
@@ -945,6 +945,10 @@ def name(self):
         """The name of the object."""
         return self._name
 
+    @property
+    def _C_type_qualifier(self):
```

Comment:
Any reason why we don't want it in `CodeSymbol` with the rest?

Reply:
right, thanks